packages/flutter/lib/src/material/stepper.dart

```diff
@@ -12,6 +12,7 @@ import 'icons.dart';
 import 'ink_well.dart';
 import 'material.dart';
 import 'material_localizations.dart';
+import 'scroll_physics.dart';
```

Comment:
This line should be removed - I'm pretty sure what you want is already handled by the import on line 5.

Reply:
Yeah you're right. I'll remove it.